src/content/en/showcase/2017/matterport.md

```diff
@@ -0,0 +1,36 @@
+project_path: /web/_project.yaml
+book_path: /web/showcase/_book.yaml
+
+{# wf_published_on: 2017-02-02T18:00:00.000Z #}
+{# wf_updated_on: 2017-02-02T18:00:00.000Z #}
+{# wf_featured_image: /web/showcase/2017/images/matterport/VR_FourSeason_InGame01.jpg #}
+{# wf_featured_snippet: Matterport VR lets you experience thousands of real-world places in interactive virtual reality. Using a combination of automated 3D interior mapping technology and robust vision processing algorithms, Matterport’s 3D media system enables anyone to create immersive virtual environments from real places. Unlike 360˚ photos, Matterport Spaces allow users to interactively walk through any location.<br><br>With WebVR, anyone can <a href="https://matterport.com/virtual-reality/webvr/">explore Matterport’s library</a> of over 300,000 VR Spaces with a WebVR-enabled browser. Tour everything from celebrity homes to museums, canyons, Star Trek sets, and beyond. #}
+{# wf_tags: webvr,casestudy #}
+
+# Matterport VR for WebVR {: .page-title }
+
+<img src="/web/showcase/2017/images/matterport/VR_FourSeason_InGame01.jpg" class="attempt-right">
+
+### TL;DR {: .hide-from-toc }
+By adding WebVR support, Matterport makes it even easier to explore real-world places in immersive virtual reality. With no apps to download, WebVR will reduce drop-offs and enable partners to retain the user on their own website. [WebVR is a key lever in Matterport’s strategy](https://matterport.com/virtual-reality/webvr/) for increasing consumption and discovery of VR content.
```

Comment:
We have the same URL in this article twice. Could we include the URL right at the top with some text (try it out here or something) and then remove it from the other two locations (TLDR and Solution)?